package.json

```diff
@@ -8,14 +8,17 @@
     "build": "gulp",
     "build:dist": "gulp --dist",
     "start": "nodemon dev/server/server.js",
-    "server:dist": "node dist/server/server.js"
+    "server:dist": "node dist/server/server.js",
+    "test": "./node_modules/.bin/mocha --compilers js:babel-register --bail --reporter nyan ./src/specs/**/*.js"
```

Comment:
Yes, it will use the locally installed version. I too avoid global installs and always opt for saving to dev dependencies.